### PR TITLE
Add Hilux Dimmer-Switch-ZB3.0

### PR DESCRIPTION
**Pull request by bjornbak**
Same DIN Dimmer Switch https://www.ledproff.dk/zigbee-produkter/21936-hilux-zigbee-din-skinne-lysdaemper-250w-5714255007350.html sold under multiple Vendor names.

Previous batches I bought of it was recognized as Candeo HK-DIM-A but the latest was unsupported until I added it as Vendor Hilux too.

### CODE REVIEW

File: src/devices/hilux.ts [hunk 1]
```diff
@@ -9,6 +9,13 @@ const definitions: Definition[] = [
         description: 'Spot 7W',
         extend: [light({colorTemp: {range: [153, 370]}, powerOnBehavior: false})],
     },
+    {
+        zigbeeModel: ['Dimmer-Switch-ZB3.0'],
+        model: 'Dimmer-Switch-ZB3.0',
```
Comment (Koenkk) [inline src/devices/hilux.ts:14]: Could you update this with the actual model number written on the device?

Comment (bjornbak) [reply src/devices/hilux.ts:14]: Unfortunately it is installed by a certified electrician a while back and just didn't get around to fix in z2m before now.

I can't see any model number 😭

Comment (Koenkk) [reply src/devices/hilux.ts:14]: Is there anything written on the site of this product?

Comment (bjornbak) [reply src/devices/hilux.ts:14]: See the pictures here: https://ledpaerer.dk/zigbee-din-rail-dimmer.html?gad_source=1&gclid=CjwKCAjwy8i0BhAkEiwAdFaeGDe8224f_XdUwuEM_axNunYCiFmEvnXcBzuXeaY28oevcG6o5JkiVRoCxGUQAvD_BwE

I can't see check now that it is installed. I remember the diagram but don't remember if anything was written on the opposite side or the back.

Comment (Koenkk) [reply src/devices/hilux.ts:14]: On the site I see `D160-ZG`

File: src/devices/hilux.ts [hunk 2]
```diff
@@ -9,6 +9,13 @@ const definitions: Definition[] = [
         description: 'Spot 7W',
         extend: [light({colorTemp: {range: [153, 370]}, powerOnBehavior: false})],
     },
+    {
+        zigbeeModel: ['Dimmer-Switch-ZB3.0'],
```
Comment (Koenkk) [inline src/devices/hilux.ts:13]: Could you provide the data/database.db of this device? There are multiple devices which use the model ID so we should use a `fingerprint` here (example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/candeo.ts#L13)

Comment (bjornbak) [reply src/devices/hilux.ts:14]: Is https://github.com/Koenkk/zigbee-herdsman-converters/pull/7763/commits/19e83eb3af83cc6f3267b6f0003fe21c6cff09c5 what you are asking for?

Comment (Koenkk) [reply src/devices/hilux.ts:14]: Yes, can you also try if it works after removing `zigbeeModel: ['Dimmer-Switch-ZB3.0'],`?

Comment (bjornbak) [reply src/devices/hilux.ts:14]: Just to make sure we are on the same page.
What I am testing is 

```
external_converters:
  - hilux.js
```
in configuration.yaml

hilux.js:
```
const {light} = require('zigbee-herdsman-converters/lib/modernExtend');

const definition = {
    fingerprint: [{modelID: 'Dimmer-Switch-ZB3.0', manufacturerName: 'Hilux'}],
    model: 'Dimmer-Switch-ZB3.0',
    vendor: 'Hilux',
    description: 'Zigbee LED dimmer smart switch',
    extend: [light({configureReporting: true})],
};

module.exports = definition;
```

Comment (Koenkk) [reply src/devices/hilux.ts]: Perfect, thanks!

**Comment by Koenkk** [comment]
Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `D160-ZG.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Pull request merged**